app/uk/gov/hmrc/minorentityidentificationfrontend/views/overseasCompanyViews/capture_overseas_tax_identifiers_country_page.scala.html

```diff
@@ -0,0 +1,90 @@
+@*
+ * Copyright 2022 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *@
+
+@import uk.gov.hmrc.govukfrontend.views.html.components._
+@import uk.gov.hmrc.govukfrontend.views.html.components
+@import uk.gov.hmrc.hmrcfrontend.views.html.components.implicits._
+@import uk.gov.hmrc.minorentityidentificationfrontend.config.AppConfig
+@import uk.gov.hmrc.minorentityidentificationfrontend.models.PageConfig
+@import uk.gov.hmrc.minorentityidentificationfrontend.views.helpers.TitleHelper.title
+@import uk.gov.hmrc.minorentityidentificationfrontend.views.html.helpers._
+@import uk.gov.hmrc.minorentityidentificationfrontend.views.html.templates.govuk_template
+@import uk.gov.hmrc.minorentityidentificationfrontend.controllers.overseasControllers.routes
+@import uk.gov.hmrc.minorentityidentificationfrontend.models.Country
+
+@this(layout: govuk_template,
+        h1: h1,
+        govukButton: GovukButton,
+        formWithCSRF: FormWithCSRF,
+        govukInput: GovukInput,
+        govukErrorSummary: GovukErrorSummary,
+        link: link,
+        select: GovukSelect
+)
+
+@(journeyId: String, pageConfig: PageConfig, formAction: Call, form: Form[_], countries: Seq[Country])(implicit request: Request[_], messages: Messages, appConfig: AppConfig)
+
+@layout(pageConfig, title = title("tax-identifiers-country.title", form)) {
+
+    @if(form.errors.nonEmpty) {
+        @govukErrorSummary(ErrorSummary().withFormErrorsAsText(form))
+    }
+
+    @h1(messages("tax-identifiers-country.title"))
+
+    @formWithCSRF(action = formAction) {
+
+        @select(components.Select(
+            id = "country",
+            name = "country",
+            label = Label(content = HtmlContent(messages("tax-identifiers-country.title")), classes = "govuk-visually-hidden"),
```

Comment:
Just checking do we really want the label to be "Which country issued the overseas tax identifier?"?

Reply:
I was getting accessibility issue without label so added and hide it. Please see below
![Screenshot 2022-08-04 at 15 50 12](https://user-images.githubusercontent.com/75069573/182877707-ee2d066b-adee-44e9-8a9d-3431a557973a.png)